cmd/nsm-init/nsm-init.go

```diff
@@ -231,21 +215,45 @@ func main() {
 		}
 	}
 	logrus.Infof("nsm client: %d NetworkServices discovered from Local NSM.", len(availablaNetworkServices))
-	if err := applyRequiredConfig(ns); err != nil {
-		logrus.Fatalf("nsm client: initialization failed, exiting...", err)
+
+	// For NSM to program container's dataplane, container's linux namespace must be sent to NSM
+	linuxNS := getCurrentNS()
+	if linuxNS == "" {
+		logrus.Fatal("nsm client: failed to get a linux namespace for the current process, exiting...")
 		os.Exit(1)
 	}
+
+	// TODO (sbezverk) At this point it is not clear the logic what to do if nsm init process is requesting
```

Comment:
My personal preference here is that we loop through and connect each one. I could see letting the user specify which ones were "stop running if this connection fails" vs. "it's ok if this one doesn't work, just let me know", in which case we'd act accordingly.

Reply:
Ok, sounds good, I will play with possible solutions

Reply:
👍